web/webpack.sw.config.js

```diff
@@ -2,6 +2,7 @@ const NODE_ENV = process.env.NODE_ENV || 'development';
 
```

Comment:
There is an additional `sw.js.LICENSE.txt` file, which references Google LLC.
Do you know why this is happening -- are we using some Google open source stuff that requires relaying this license.txt file?

Otherwise, people might think our PWA or company is related to Google again.

Reply:
Hm ... most likely it's the open source thing. Here's the ui.js one for comparison
- [sw.js.LICENSE.txt](https://github.com/OdyseeTeam/odysee-frontend/files/9429935/sw.js.LICENSE.txt)
- [ui-202208260441.324b17f302.js.LICENSE.txt](https://github.com/OdyseeTeam/odysee-frontend/files/9429937/ui-202208260441.324b17f302.js.LICENSE.txt)

Reply:
Looking at the production build from `master`, it looks like webpack 5 is factoring out the license text from the file.

See here: https://odysee.com/sw.js

![image](https://user-images.githubusercontent.com/128739/188251444-77c70e8f-8b5c-483e-be46-b18d1ed0f2c2.png)
kons

And looks to have a 30k savings there:

![image](https://user-images.githubusercontent.com/128739/188251468-a4e40f74-6026-4741-b645-d9331b96c6ab.png)


Optionally, the comments (which would include the license info) can be removed entirely with an option to `TerserPlugin` as well.

Reply:
As for what from google is being used, it appears that there is some firebase integration. I'd imagine you guys are using it for something like your notifications? I haven't dug into the code to see what. But there initialization is there.

Reply:
Ah, yes.  Everything makes sense now.